velox/functions/prestosql/tests/MapFromEntriesTest.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/prestosql/tests/utils/FunctionBaseTest.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::functions::test;
+
+namespace {
+std::optional<std::vector<std::pair<int32_t, std::optional<int32_t>>>> O(
+    const std::vector<std::pair<int32_t, std::optional<int32_t>>>& vector) {
+  return std::make_optional(vector);
+}
+} // namespace
+
+namespace {
+class MapFromEntriesTest : public FunctionBaseTest {
+ protected:
+  void verifyMapFromEntriesWithTry(
+      const VectorPtr& input,
+      const VectorPtr& expected) {
+    verifyMapFromEntries(input, expected, true);
+  }
```

Comment:
nit: No need to add a function with a single-line body. You can just call verifyMapFromEntries(input, expected, true) in the caller. Ditto for evaluateExpr.

Reply:
I agree with calling verifyMapFromEntries() instead. 
For evaluateExpr(), I think it's necessary to encapsulate the std::vector<VectorPtr> to RowVectorPtr conversion in a utility function, which is consistent with other utility functions in this class. See all the call sites of makeRowVector().